src/platform_impl/macos/monitor.rs

```diff
@@ -158,6 +158,27 @@ pub fn primary_monitor() -> MonitorHandle {
   MonitorHandle(CGDisplay::main().id)
 }
 
+// `from_point` get a monitor handle which contains the given point.
+pub fn from_point(x: f64, y: f64) -> Option<MonitorHandle> {
+  unsafe {
+    for monitor in available_monitors() {
+      let bound = CGDisplayBounds(monitor.0);
+      // dumb way. a better way is to use [CGRectContainsPoint](https://developer.apple.com/documentation/coregraphics/1456316-cgrectcontainspoint)
+      // which is not supported by core-graphics binding yet.
+      // Once the [PR](https://github.com/servo/core-foundation-rs/pull/526) is accepted
+      // we can use that method.
+      if x >= bound.origin.x
+        && x <= bound.origin.x + bound.size.width
+        && y >= bound.origin.y
+        && y <= bound.origin.y + bound.size.height
+      {
+        return Some(monitor);
+      }
+    }
+  }
+  return None;
+}
+
```

Comment:
Windows and Linux have a dedicated API for these and I am not sure if macOS has one or not but if it doesn't, this is fine.

- Linux: https://gtk-rs.org/gtk3-rs/stable/latest/docs/gdk/struct.Display.html#method.monitor_at_point
- Widnows: https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-monitorfrompoint

Reply:
Thanks for your suggestion. I will use these two method for windows and linux